.travis.yml

```diff
@@ -10,6 +10,7 @@ script:
   - npm run test
 deploy:
   provider: script
+  skip_cleanup: true
```

Comment:
Why are we skipping cleanup?

Reply:
If you don't, then Travis cleans up the working folder, including the artifacts of the previous build.  The package.json doesn't specify a prepublishOnly step, which could allow the build to run again, but since the build runs in an earlier step, that seemed unnecessary.

https://docs.travis-ci.com/user/deployment#uploading-files-and-skip_cleanup

Without either skipping cleanup or running the build again, you get an empty package:

```
npm notice 
npm notice 📦  demux@3.1.3-cb24e31.0
npm notice === Tarball Contents === 
npm notice 1.4kB   package.json      
npm notice 51B     .npmrc.template   
npm notice 6B      .nvmrc            
npm notice 278B    CONTRIBUTING.md   
npm notice 6.2kB   README.md         
npm notice 87B     run-example.sh    
npm notice 73B     typings.d.ts      
npm notice 253.8kB yarn.lock         
npm notice 1.3kB   scripts/publish.sh
npm notice === Tarball Details === 
npm notice name:          demux                                   
npm notice version:       3.1.3-cb24e31.0                         
npm notice package size:  98.8 kB                                 
npm notice unpacked size: 263.2 kB                                
npm notice shasum:        ee0677f0cb53a518763f002cfcdc11d2c6e9da71
npm notice integrity:     sha512-z/j/KfCzfy2+N[...]qP9W/IHK6QADg==
npm notice total files:   9                                       
npm notice 
+ demux@3.1.3-cb24e31.0
```